cms/static/cms/js/modules/cms.toolbar.js

```diff
@@ -649,6 +649,12 @@ var CMS = window.CMS || {};
                             message: el.data('text')
                         });
                         break;
+                    case 'cms_frame':
+                        var cms_frame = new CMS.CMSFrame();
+                        cms_frame.open({
```

Comment:
wouldn't it be just enough to use ``window.open(el.attr('href'), 'cms_frame')`` here to avoid an extra file (would be to much overkill IMO) like:

case 'cms_frame':
    frame = window.open(url, 'cms_frame');
    frame.focus();